Test-NetStack.psm1

```diff
@@ -891,6 +891,8 @@ Function Test-NetStack {
             $NetStackHelperModules = Get-ChildItem (Join-Path -Path $PSScriptRoot -ChildPath 'Helpers\*') -Include '*.psm1'
             $NetStackHelperModules | ForEach-Object { $ISS.ImportPSModule($_.FullName) }
 
+            Get-VDiskStatus($LogFile)
```

Comment:
What happens if there is no vDisk? This may not be an S2D system.

Reply:
Would it make sense to just exit the stage in that case?

Reply:
We may need to discuss as i'm not sure about the differences in the stage from the general (public) stages. Generally that's not how we handle the stages. The stage path status should be failed.

Reply:
We may need to discuss. Generally that's not how we handle the stages. The stage path status should be failed.

This seems like a dependency for the stage to run. We may want to review earlier and abort the stage (should be an example in previous stages).

Reply:
Spoke to Travis about this, he said it is fine if the stage 7 is treated as if it will only be used on S2D systems, and that it should be treated as an experimental stage not to be used by everyone.